src/Protocol/Diaspora.php

```diff
@@ -4135,8 +4135,7 @@ private static function createProfileData($uid)
 				$dob = DateTimeFormat::utc($year . '-' . $month . '-'. $day, 'Y-m-d');
 			}
 
-			$about = $profile['about'];
-			$about = strip_tags(BBCode::convert($about));
+			$about = BBCode::toPlaintext($profile['about']);
```

Comment:
I can imagine that they are using Markdown for that field.

Reply:
Indeed, I supposed that we knew what we were doing with this.

Reply:
And I just tested. Diaspora is indeed using markdown there.

Reply:
Thanks for confirming, I added the required change yesterday.

Reply:
I just wanted to test this behaviour.